src/state/connection-mixin.ts

```diff
@@ -211,6 +211,7 @@ export const connectionMixin = <T extends Constructor<HassBaseEl>>(
         formatEntityAttributeName: (_stateObj, attribute) => attribute,
         formatEntityAttributeValue: (stateObj, attribute, value) =>
           value != null ? value : (stateObj.attributes[attribute] ?? ""),
+        formatEntityAttributeUnit: () => "",
```

Comment:
It's already `string | undefined` so better to return `undefined` I think
```suggestion
        formatEntityAttributeUnit: () => undefined,
```

Reply:
Agree